apps/openchallenges/challenge-service/src/main/java/org/sagebionetworks/openchallenges/challenge/service/service/ChallengeAnalyticsService.java

```diff
@@ -0,0 +1,23 @@
+package org.sagebionetworks.openchallenges.challenge.service.service;
+
+import java.util.Arrays;
+import java.util.List;
+import org.sagebionetworks.openchallenges.challenge.service.model.dto.ChallengesPerYearDto;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class ChallengeAnalyticsService {
+
+  @Transactional(readOnly = true)
+  public ChallengesPerYearDto getChallengesPerYear() {
+    List<String> years =
+        Arrays.asList(
+            "1969", "2011", "2012", "2013", "2014", "2015", "2016", "2017", "2018", "2019", "2020",
```

Comment:
Thanks for feeding static challenge counts per year. 
`1969` year is suspicious, so I realize it was a miscalculation caused by undefined dates. I will look into this.

Reply:
### Updated data
```json
{
    "years": [
        "2011",
        "2012",
        "2013",
        "2014",
        "2015",
        "2016",
        "2017",
        "2018",
        "2019",
        "2020",
        "2021",
        "2022",
        "2023"
    ],
    "challengeCounts": [
        1,
        6,
        4,
        6,
        7,
        9,
        25,
        11,
        19,
        18,
        42,
        22,
        18
    ]
}
```

Reply:
Fixed